rpcserver.go

```diff
@@ -1623,6 +1639,78 @@ func extractOpenChannelMinConfs(in *lnrpc.OpenChannelRequest) (int32, error) {
 	}
 }
 
+// extractSendCoinsMinConfs extracts the minimum number of confirmations from
+// the SendCoinsRequest that each output used to fund the transaction should
+// satisfy.
+func extractSendCoinsMinConfs(in *lnrpc.SendCoinsRequest) (int32, error) {
```

Comment:
we can share this logic by passing in `MinConfs and `SpendUnconfirmed` instead of the RPC specific request.

Reply:
Yeah, this is what I meant by "needs consolidation" :)... Can you suggest where this more general utility method should live in the codebase?

Reply:
Extracted to more generalised `lnrpc.ExtractMinConfs` utility method.